Lexicon.md

```diff
@@ -4,19 +4,20 @@
 * __Bit String__: A sequence of __Bits__ of arbitrary length $$ B^n := (b_k)_{k=0}^n , \  k \in \mathbb{N} $$.
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
 * __Information__: Anything that can be encoded into a __Bit String__.
-* __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __Environment__: __Information__ that is defined relative to a particular __Actor__. It is seperated from an __Actor__ by an __Actor Boundary__, and encompasses all content needed to process any relevant __Events__.
 * __World__: Synonymous with __Environment__.
-* __Agent__: A __System__ that interacts with an __Environment__. It can consume zero or more __Observations__, and perform zero or more __Actions__. The decision of which __Actions__ to perform is made based on the __Agent's__ beliefs.
-* __Agent Boundary__: The surface that separates an __Agent__ from its __Environment__.
-* __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
-* __Observation Space__: The set of all __Observations__ that a particular __Agent__ may observe. All __Observations__ perceived by an __Agent__ are sampled from its __Observation Space__.
+* __Actor__: __Information__ that interacts with an __Environment__ through __Events__.
+* __Actor Boundary__: The surface that separates an __Actor__ from its __Environment__.
+* __Event__: __Information__ that crosses the __Actor Boundary__.
+* __Observation__: An __Event__ in which __Information__ enters an Agent.
+* __Observation Space__: The set of all __Observations__ that a particular Agent may observe. All __Observations__ perceived by an Agent are sampled from its __Observation Space__.
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
 * __Model__: A __System__ that represents another __System__. It can be assigned a truth value (measurement) which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
 * __World State__: Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant.
-* __World Model__: An __Agent__'s internal __Model__ of its __World__.
```

Comment:
I'd like to flag this for future discussion when we re-address #28. Can an __Actor__ have a __World Model__ or is that a privilege of an Agent?

Reply:
I would say only agents have internal models. Still unclear on what counts as an actor.

Reply:
Based on my understanding, an Actor is pretty much any way of organizing Information. So a rock is an actor, a napkin is an actor. A human is an actor. I'm thinking of actors as a super set of agents. All Agents are Actors, but not all Actors are Agents.

We still haven't defined what makes an Agent an Agent. One of the suggestions you made previously is that an Agent is an Actor that possesses a world model. Another option is an Agent is an Actor with high Solomonoff complexity in the relationship between its Actions and Observations.

Reply:
Agreed that Agents are a subset of Actors. Are Actors a subset of Objects?

Reply:
Having them be a subset of Objects makes us fall into a definition chain where an Object is  a set of correlated Observations, and Observations are Information passing into an Agent. Then an Actor could not exist without an Agent observing it. 

Furthermore if Actors are a superset of Agents, and Agent would be defined in terms of Observations as well (which is a cyclic definition).

I'm not sure what to make of that. What do you think?

Reply:
I think it is consistent with logical relativism. All objects are models inferred from observations, the models are internal to agents. We assume that at least some of the models are causally correlated with corresponding things in the environment but we shouldn't mistake the maps (models) for the territory (environment).

Reply:
In #28 I [wrote](https://github.com/Vulcan-Academy/Vulcanomicon/issues/28#issuecomment-350507298):
> I'd like to define agent as a system that fits the definition of philosophical agency meaning something that acts which I think we can equate with changing the state of the world in an intentional way. 

That seems to imply that all actors are agents or we have to change how to interpret what it means to act.

Reply:
> I think it is consistent with logical relativism. All objects are models inferred from observations, the models are internal to agents. 

I can see how this can be confusing. All models are representations, i.e. patterns that represent other patterns. All we have access to are models, even our subjective experience. But because good (accurate, useful) models tend to be highly correlated with what they represent we tend to forget they are models and drop the distinction between signifier and signified.

Reply:
Ok this makes sense, I agree with defining everything in terms of models. The only thing I'd like to tease out with our current lexicon is whether Information, then, is a Model?

Reply:
My intuition is no. 

Would you agree with the following generalization?

Our entry point into the lexicon is Information. The rest of the lexicon relates to information making sense of other information. The Agent Architecture (the definitions surrounding Agents, Actors, Actions, Observations, etc...) are a Model of how Information can reason about other Information.

Reply:
I agree with the generalization except to say that Information is a perfectly good entry point but maybe not the only one.

Reply:
Ok great. I agree "entry point" may have not been the best way to phrase that. Maybe "Information is the lowest level of abstraction", or "Information is the foundation of the lexicon".